### DISCUSSION TITLE
Just seeing if I can wake Nota up....

### DISCUSSION THREAD

**Discussion by marlonjames71** [notaapp/talk#432] [Random]
Hello there 👋📯

**Comment by nelsonfigueroa** [comment]
I think this project has been abandoned 😕 but I do hope you get their attention.

**Reply by Arbiter313** [reply]
That would be a shame, I think any communication from the developers would be appreciated

**Comment by erusev** [comment]
We'll try to release an update later this month. Won't be a huge release, but it should bring some nice improvements.
